### PR TITLE
DOC-3455: Refine the doc website menu

### LINKED ISSUE #3455: Improve menus for new databend.rs website
**Summary**

1. menu improvement

### PR DESCRIPTION
**Pull request by BohuTANG**
I hereby agree to the terms of the CLA available at: https://databend.rs/policies/cla/

## Summary

Refine the doc website menu

## Changelog

- Documentation

## Related Issues

Fixes #3455

## Test Plan

Unit Tests

Stateless Tests

### CODE REVIEW

**Comment by BohuTANG** [comment]
The menu seems ok now, there are some TODOs:
1. The black index.md, prefered https://docusaurus.io/docs/category/guides
2. Change the domain to the new website
cc @Xuanwo

File: website/databend/docusaurus.config.js [hunk 1]
```diff
@@ -114,14 +111,14 @@ const config = {
                     {
                         title: 'Resources',
                         items: [
+                            {
+                                label: 'CLI Reference',
+                                to: '/user/cli'
```
Comment (Xuanwo) [inline website/databend/docusaurus.config.js:116]:
```suggestion
                                to: '/user/cli/index'
```

Comment (BohuTANG) [reply website/databend/docusaurus.config.js]: Thanks, done.

File: website/databend/docs/dev/development/_category_.yml [hunk 2]
```diff
@@ -0,0 +1 @@
+label: 'Policies'
```
Comment (Xuanwo) [inline website/databend/docs/dev/development/_category_.yml:1]:
```suggestion
label: 'Development'
```

File: website/databend/docs/dev/policies/_category_.yml [hunk 3]
```diff
@@ -0,0 +1 @@
+label: 'Development'
```
Comment (Xuanwo) [inline website/databend/docs/dev/policies/_category_.yml:1]:
```suggestion
label: 'Policies'
```

**Comment by Xuanwo** [comment]
> 1. The black index.md, prefered https://docusaurus.io/docs/category/guides

Tracked in https://github.com/datafuselabs/databend/issues/3578

**Review by Xuanwo — APPROVED** [review]

**Review by PsiACE — APPROVED** [review]

**Pull request merged**